### PR TITLE
Add Ruby Client for Dialogflow API section

### PR DESCRIPTION
**Pull request by vitaliiprodan**
Why:

* Provide example of gem usage to make developers life easier

### CODE REVIEW

File: guides/dialogflow/ruby-client-for-dialogflow-api.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Ruby Client for Dialogflow API
+
+Google cloud provides [ruby client for Dialogflow API](https://github.com/googleapis/google-cloud-ruby/tree/master/google-cloud-dialogflow). Before using this gem you may need to [create an agent and get special credentials](../agents-management.md).
```
Comment (alvaro-chalar) [inline guides/dialogflow/ruby-client-for-dialogflow-api.md]: `Ruby Client`

File: guides/dialogflow/ruby-client-for-dialogflow-api.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+`gem 'google-cloud-dialogflow'`
+
+Then create `config/initializers/google_cloud_dialogflow.rb`
+
+`require 'google/cloud/dialogflow/v2'`
+
+## Gem Usage
+
+This is the example of the class which allows developers to detect intent by text or event. It requires following arguments:
```
Comment (alvaro-chalar) [inline guides/dialogflow/ruby-client-for-dialogflow-api.md]: This is `a class example` which...... It requires `the` following arguments:

File: guides/dialogflow/ruby-client-for-dialogflow-api.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+- `keyfile_json` - content of keyfile.json file. See [this article](../agents-management.md) to check how to get it
+- `parameters` - hash of configurations. It allows to set following parameters:
```
Comment (alvaro-chalar) [inline guides/dialogflow/ruby-client-for-dialogflow-api.md]: It allows `you` to set `the` following parameters

File: guides/dialogflow/ruby-client-for-dialogflow-api.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    def sessions_client
+      @sessions_client = Google::Cloud::Dialogflow::V2::SessionsClient.new(
+        credentials: keyfile
+      )
+    end
+  end
+end
+```
+
+Both methods `detect_text_intent` and `detect_event_intent` will return instance of [`Google::Cloud::Dialogflow::V2::DetectIntentResponse`](https://googleapis.github.io/google-cloud-ruby/docs/google-cloud-dialogflow/latest/Google/Cloud/Dialogflow/V2/DetectIntentResponse.html). So to response parameters, for example `whisper_message` you can use this:
```
Comment (alvaro-chalar) [inline guides/dialogflow/ruby-client-for-dialogflow-api.md]: So, to `get`  response parameters such as `whisper_message`, you can use:

**Comment by vitaliiprodan** [comment]
@alvaro-chalar that's fixed

**Review by alvaro-chalar — APPROVED** [review]

**Review by Ammet — APPROVED** [review]

**Pull request merged**